org.audacityteam.Audacity.yaml

```diff
@@ -185,9 +185,9 @@ modules:
       - -DRAPIDJSON_BUILD_TESTS=OFF
       - -DRAPIDJSON_BUILD_THIRDPARTY_GTEST=OFF
     sources:
-      - type: archive
-        url: https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz
-        sha256: bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
+      - type: git
+        url: https://github.com/miloyip/rapidjson.git
+        commit: d621dc9e9c77f81e5c8a35b8dcc16dcd63351321
```

Comment:
I'd rather have the patches that fix the build instead rather than the random revisions.

Reply:
like https://github.com/flathub/org.freedesktop.LinuxAudio.Plugins.Geonkick/blob/branch/24.08/patches/rapidjson-gcc14.patch